lib/src/main/java/com/metaplex/lib/experimental/serialization/serializers/solana/AnchorAccountSerializer.kt

```diff
@@ -0,0 +1,40 @@
+/*
+ * AnchorAccountSerializer
+ * Metaplex
+ * 
+ * Created by Funkatronics on 7/25/2022
+ */
+
+package com.metaplex.lib.experimental.serialization.serializers.solana
+
+import kotlinx.serialization.KSerializer
+import kotlinx.serialization.descriptors.SerialDescriptor
+import kotlinx.serialization.encoding.Decoder
+import kotlinx.serialization.encoding.Encoder
+import kotlinx.serialization.serializer
+import java.nio.ByteBuffer
+import java.nio.ByteOrder
+import java.nio.charset.StandardCharsets
+import java.security.MessageDigest
+
+class AnchorAccountSerializer<T>(accountName: String, serializer: KSerializer<T>)
```

Comment:
Not all account responses are anchor accounts. How should we discriminate without specifying the serializer?

For example if we are deciding an nft metaplex account I doesn't contain that and the object @serializable doesn't provide enough information about the discriminator.

@anchorSerializable?

Reply:
Ha you are thinking exactly what I was thinking! I am trying to create a custom annotation, @AnchorAccount or something that would then tell the system to use that serializer.